lib/model.php

```diff
@@ -76,10 +80,18 @@ public function purge()
      */
     private function _getStore()
     {
+        // FIXME
+        // Workaround so that config value don't need to be changed
```

Comment:
It's totally up to you. Righ, I forgot to mention that change in the description.